ssg/templates.py

```diff
@@ -19,6 +19,29 @@
 # Callback functions for processing template parameters and/or validating them
 
 
+def mount_option(data, lang):
+    if lang != "oval":
+        data["mountoption"] = re.sub(" ", ",", data["mountoption"])
+    else:
+        data["pointid"] = re.sub(r"[-\./]", "_", data["mountpoint"]).lstrip("_")
+    return data
+
+
+def mount_option_special(data, lang):
+    data = mount_option(data, lang)
+    if lang != "oval":
+        data["pointid"] = re.sub(r"^var_(.*)", r"\1s", data["mountpoint"])
+    return data
+
+
+def mount_option_var(data, lang):
```

Comment:
This function is unused, is it expected?

Reply:
Yes, I remove it.